### PR TITLE
Update LICENSE file as MIT

### PR DESCRIPTION
**Pull request by oreno4649**
I update LICENSE file as MIT.

Please check the license notation.

https://github.com/graphql/graphql.github.io/issues/438

### CODE REVIEW

**Comment by ucan-lab** [comment]
Please Merge ! 🙏

File: LICENSE [hunk 1]
```diff
@@ -1,26 +1,22 @@
...
+The MIT License (MIT)
 
-Facebook, Inc. (“Facebook”) owns all right, title and interest, including all
-intellectual property and other proprietary rights, in and to the graphql.org
-software. Subject to your compliance with these terms, you are hereby granted a
-non-exclusive, worldwide, royalty-free copyright license to (1) use and copy the
-graphql.org software; and (2) reproduce and distribute the graphql.org software
-as part of your own software (“Your Software”). Facebook reserves all rights not
-expressly granted to you in this license agreement.
+Copyright 2015 <Facebook, Inc.>
```
Comment (carolstran) [inline LICENSE:3]: Shouldn't this be `Copyright © 2020 The GraphQL Foundation` (or something similar)?

**Review by carolstran — COMMENTED** [review]
Added a comment about the copyright. Once that's either updated or clarified - and based on the discussions in #438 and #823 - it seems like it's long overdue for merging 🚀 

Pinging @IvanGoncharov and @orta.

**Comment by IvanGoncharov** [comment]
@oreno4649 Thanks for taking the time to prepare that PR 👍 
It's a complicated issue tracked in #823 + we need to update copyright headers in all JS files.

**Pull request closed without merging**